bedrock/base/templates/base-protocol.html

```diff
@@ -141,6 +141,11 @@
         {{ js_bundle('ui') }}
         {{ js_bundle('fxa') }}
         {{ js_bundle('data') }}
+        {% if switch('m24-navigation-and-footer') %}
+          {{ js_bundle('m24-navigation') }}
+        {% else %}
+          {{ js_bundle('navigation') }}
+        {% endif %}
```

Comment:
Suggestion: rather than create a new separate JS bundle just for the navigation, could we instead create a `ui-refresh` bundle? Even thought we have HTTP/2, we still need to try and strike a performance balance, especially when it comes to global components in the base template.